### PR TITLE
SG-21557 integration admin guide

### PR DESCRIPTION
**Pull request by zamu5**
Jeff, all the images are in a bucket in S3, let me know if it's necessary download all of them and save it in the repo.

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/97188ccfeb6b8cb23e395020f72b866df5d96e49/index.html)

File: docs/en/guides/pipeline-integrations/administration/integrations-admin-guide.md [hunk 1]
```diff
@@ -5,4 +5,695 @@ pagename: integrations-admin-guide
...
+
+#### Basing your new project on an existing project
+
+![](images/Integration-admin-guide/wizard_project_config.png)
+
+This is a quick and convenient way to get up and running with a new project with all the defaults and settings that you had in a previous project. Toolkit will simply copy across the configuration from your old setup to the new project. This is a simple and pragmatic way to evolve your configuration - each new project is based on an older project.
+
+For more ways and documentation on how to evolve and maintain your pipeline configuration, see here:
+
+[![](https://sg-toolkit.s3.amazonaws.com/tank/docs/docs/53/no_version_1207_013348/icon_256.png)](https://support.shotgunsoftware.com/hc/en-us/articles/219033168#Inheriting%20the%20config%20from%20your%20previous%20project) [> Managing your project configuration.](https://support.shotgunsoftware.com/hc/en-us/articles/219033168#Inheriting%20the%20config%20from%20your%20previous%20project)
```
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/integrations-admin-guide.md:628]:
```suggestion
[Managing your project configuration](https://support.shotgunsoftware.com/hc/en-us/articles/219033168#Inheriting%20the%20config%20from%20your%20previous%20project)
```

File: docs/en/guides/pipeline-integrations/administration/integrations-admin-guide.md [hunk 2]
```diff
@@ -5,4 +5,695 @@ pagename: integrations-admin-guide
...
+
+#### Using a configuration template from git
+
+![](images/Integration-admin-guide/wizard_git.png)
+
+Use this option if you want to keep your project's configuration connected to source control. Specify a url to a remote git or github repository and the setup process will clone it for you. Note that this is not just github, but works with any git repository. Just make sure that the path to the repository ends with  `.git`, and Toolkit will try to process it as a git setup. Because your project configuration is a git repository, you can commit and push any changes you make to your master repository and beyond that to other projects. Using a github based configuration makes it easy to keep multiple Toolkit projects in sync. You can read more about it here:
+
+[![](https://sg-toolkit.s3.amazonaws.com/tank/docs/docs/53/no_version_1207_013348/icon_256.png)](https://support.shotgunsoftware.com/hc/en-us/articles/219033168#A%20studio%20configuration%20in%20git%20source%20control) [> Managing your project configuration.](https://support.shotgunsoftware.com/hc/en-us/articles/219033168#A%20studio%20configuration%20in%20git%20source%20control)
```
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/integrations-admin-guide.md:636]:
```suggestion
[Managing your project configuration](https://support.shotgunsoftware.com/hc/en-us/articles/219033168#A%20studio%20configuration%20in%20git%20source%20control)
```

File: docs/en/guides/pipeline-integrations/administration/integrations-admin-guide.md [hunk 3]
```diff
@@ -5,4 +5,695 @@ pagename: integrations-admin-guide
...
+linux:   /mnt/software/shotgun/golden_circle
+macosx:  /servers/production/software/shotgun/golden_circle
+windows: \\prod\software\shotgun\golden_circle
+```
+
+#### What can I do once I have a configuration?
+
+Once you are up and running with your first configuration, please navigate to our 'next steps' documentation to learn more about how to configure and adjust Toolkit to better suite your studio needs:
+
+[![](https://sg-toolkit.s3.amazonaws.com/tank/docs/docs/55/no_version_0513_171143/icon_256.png)](https://support.shotgunsoftware.com/hc/en-us/articles/219040688) [> Beyond your first project](https://support.shotgunsoftware.com/hc/en-us/articles/219040688)
```
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/integrations-admin-guide.md:691]:
```suggestion
[Beyond your first project](https://support.shotgunsoftware.com/hc/en-us/articles/219040688)
```

**Review by thebeeland — APPROVED** [review]

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/77e94cc8bfb7908668bbbc5055badfba45799c55/index.html)

**Pull request merged**